### PR TITLE
Add missing dependencies on TBB

### PR DESCRIPTION
**Pull request by fwyzard**
#### PR description:

Add missing dependencies on TBB to the BuildFiles.

#### PR validation:

None.

### CODE REVIEW

**Comment by fwyzard** [comment]
please test

**Comment by fwyzard** [comment]
+heterogeneous

File: HeterogeneousCore/AlpakaInterface/BuildFile.xml [hunk 1]
```diff
@@ -1,3 +1,4 @@
 <use name="alpaka"/>
 <use name="boost_header"/>
+<use name="tbb"/>
```
[the remark this answers is not in the document]
Comment (fwyzard) [reply HeterogeneousCore/AlpakaInterface/BuildFile.xml:3]: Mhm... my mistake, I included the comments in my search for `tbb::` :-)

File: RecoTracker/CkfPattern/BuildFile.xml [hunk 2]
```diff
@@ -1,22 +1,23 @@
-<export>
-  <lib name="1"/>
-</export>
-<use name="RecoTracker/TkDetLayers"/>
-<use name="FWCore/Framework"/>
-<use name="FWCore/PluginManager"/>
-<use name="FWCore/ParameterSet"/>
-<use name="FWCore/MessageLogger"/>
+<use name="root"/>
+<use name="tbb"/>
```
[the remark this answers is not in the document]
Comment (fwyzard) [reply RecoTracker/CkfPattern/BuildFile.xml:2]: You're right... I'll leave a comment in the BuildFile.xml for the next person that relies only on `grep` to identify the missing dependencies :-)

**Comment by fwyzard** [comment]
please test

**Comment by fwyzard** [comment]
+heterogeneous

**Comment by fwyzard** [comment]
I didn't touch anything under `Alignment` or `CondFormats` ...

**Comment by fwyzard** [comment]
please test

**Comment by makortel** [comment]
Comparison differences are from 11634.7 (i.e. spurious, #39803)

**Comment by makortel** [comment]
+core

**Comment by cecilecaillol** [comment]
+l1

**Comment by civanch** [comment]
+1

**Comment by missirol** [comment]
+hlt

**Comment by swertz** [comment]
+xpog

**Comment by mandrenguyen** [comment]
+reconstruction

**Comment by srimanob** [comment]
+Upgrade

**Comment by emanueleusai** [comment]
+1

- technical PR

**Comment by makortel** [comment]
@cmsbuild, please test

Let's refresh

**Comment by makortel** [comment]
+heterogeneous

**Comment by makortel** [comment]
@cms-sw/heterogeneous-l2 Could you please review and sign?

**Comment by fwyzard** [comment]
> @cms-sw/heterogeneous-l2 Could you please review and sign?

I'm confused, didn't you just do it ?

**Comment by makortel** [comment]
> > @cms-sw/heterogeneous-l2 Could you please review and sign?
> 
> I'm confused, didn't you just do it ?

Yeah, it should have been "@cms-sw/generators-l2 Could you please review and sign". Thanks for point it out.

**Comment by perrotta** [comment]
+1
- @cms-sw/generators-l2 changes in the [only file in your area](https://github.com/cms-sw/cmssw/pull/40376/files#diff-c4de5ad635d29b0fb674dc1741b4cd6a30af1064b53bbc6063f91e8ee3fa8f7e) affected are technical and rather straightforward: let merge this PR before it gets unmergeable, and please complain later if you see anything wrong in it

**Comment by perrotta** [comment]
merge

**Pull request merged**